$ time sysbench --config-file=sysbench.conf oltp_update_index --tables=32 --table-size=1000000 --db-ps-mode=auto --rand-type=uniform run
sysbench 1.0.20 (using system LuaJIT 2.1.1700206165)

Running the test with following options:
Number of threads: 16
Report intermediate results every 10 second(s)
Initializing random number generator from current time


Initializing worker threads...

Threads started!

[ 10s ] thds: 16 tps: 3605.28 qps: 3605.28 (r/w/o: 0.00/3519.49/85.79) lat (ms,95%): 6.79 err/s: 0.00 reconn/s: 0.00
[ 20s ] thds: 16 tps: 3682.78 qps: 3682.78 (r/w/o: 0.00/3592.48/90.30) lat (ms,95%): 6.79 err/s: 0.00 reconn/s: 0.00
[ 30s ] thds: 16 tps: 3604.69 qps: 3604.69 (r/w/o: 0.00/3513.39/91.30) lat (ms,95%): 6.91 err/s: 0.00 reconn/s: 0.00
[ 40s ] thds: 16 tps: 3473.70 qps: 3473.70 (r/w/o: 0.00/3386.30/87.40) lat (ms,95%): 7.56 err/s: 0.00 reconn/s: 0.00
[ 50s ] thds: 16 tps: 3593.71 qps: 3593.71 (r/w/o: 0.00/3504.41/89.30) lat (ms,95%): 7.43 err/s: 0.00 reconn/s: 0.00
[ 60s ] thds: 16 tps: 3935.19 qps: 3935.19 (r/w/o: 0.00/3832.79/102.40) lat (ms,95%): 6.21 err/s: 0.00 reconn/s: 0.00
[ 70s ] thds: 16 tps: 752.10 qps: 752.10 (r/w/o: 0.00/732.20/19.90) lat (ms,95%): 6.43 err/s: 0.00 reconn/s: 0.00
[ 80s ] thds: 16 tps: 0.00 qps: 0.00 (r/w/o: 0.00/0.00/0.00) lat (ms,95%): 0.00 err/s: 0.00 reconn/s: 0.00
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest20 SET k=k+1 WHERE id=?'
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest6 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query
(last message repeated 1 times)
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest16 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest14 SET k=k+1 WHERE id=?'
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest20 SET k=k+1 WHERE id=?'
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest1 SET k=k+1 WHERE id=?'
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest27 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query
(last message repeated 3 times)
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest25 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest19 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest14 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest24 SET k=k+1 WHERE id=?'
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest26 SET k=k+1 WHERE id=?'
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest6 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query
(last message repeated 1 times)
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest4 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest7 SET k=k+1 WHERE id=?'
FATAL: mysql_stmt_execute() returned error 2013 (Lost connection to server during query) for query 'UPDATE sbtest23 SET k=k+1 WHERE id=?'
FATAL: `thread_run' function failed: ./oltp_common.lua:458: SQL error, errno = 2013, state = 'HY000': Lost connection to server during query

real    1m22.553s
user    0m1.254s
sys     0m2.981s



========================
========================
========================
========================

runtime: traceback stuck. pc=0x4cbc040 sp=0x3240d0708
stack: frame={sp:0x3240d0708, fp:0x3240d0708} stack=[0x3240ca000,0x3240d2000)
0x00000003240d0608:  0x00000037c8b05dc8  0x0000000001baa9f8 <runtime.(*mspan).writeHeapBitsSmall+0x0000000000000068>
0x00000003240d0618:  0x0000000001bab25c <runtime.(*mcache).refill+0x00000000000001d4>  0x0000000319f96008
0x00000003240d0628:  0x0000000009b09860  0x00000003dc67ff80
0x00000003240d0638:  0x00000003dcb4c000  0x0000000000000010
0x00000003240d0648:  0x000000063fbe1500  0x0000000000000000
0x00000003240d0658:  0x0000000000000268  0x0000000000000001
0x00000003240d0668:  0x0000000000000018  0x0000000000000008
0x00000003240d0678:  0x0000000001ba11e0 <runtime.mallocgc+0x0000000000000508>  0x0000003d99695808
0x00000003240d0688:  0x00000003dc67f800  0x00000037c8b05dc8
0x00000003240d0698:  0x00000003233bdb18  0x00000003240d0660
0x00000003240d06a8:  0x0000000004d7e694 <github.com/pingcap/tidb/pkg/server.(*clientConn).writeOK+0x0000000000000074>  0x0000003a97d3e2a1
0x00000003240d06b8:  0x00000003e0f70d80  0x0000000009b52480
0x00000003240d06c8:  0x0000000001ba15e8 <runtime.newobject+0x0000000000000038>  0x0000003f92ca4b08
0x00000003240d06d8:  0x0000000001bd0018 <runtime.freedefer+0x0000000000000280>  0x0000003b826b8fc0
0x00000003240d06e8:  0x0000003b826b8fc0  0x0000000004d88b3c <github.com/pingcap/tidb/pkg/server.(*clientConn).executePreparedStmtAndWriteResult+0x00000
00000000d44>
0x00000003240d06f8:  0x0018000100000610  0x0000000004d8b914 <github.com/pingcap/tidb/pkg/server.(*TiDBContext).ExecuteStmt+0x0000000000000064>
0x00000003240d0708: >0x0000000004cbc040 <github.com/pingcap/tidb/pkg/session.(*session).ExecuteStmt.deferwrap1+0x0000000000000000>  0x0000000009ae0fa0
0x00000003240d0718:  0x0000000000000000  0x0000000000000000
0x00000003240d0728:  0x0000000004d88254 <github.com/pingcap/tidb/pkg/server.(*clientConn).executePreparedStmtAndWriteResult+0x000000000000045c>  0x0000
0004b1d96a08
0x00000003240d0738:  0x00000000065edd88  0x00000003e0f70d80
0x00000003240d0748:  0x00000000066020d8  0x00000003dcb4c000
0x00000003240d0758:  0x0000000004d8820c <github.com/pingcap/tidb/pkg/server.(*clientConn).executePreparedStmtAndWriteResult+0x0000000000000414>  0x0000
00000000000c
0x00000003240d0768:  0x00000000054ca6c0  0x0000000000010001
0x00000003240d0778:  0x0000000004d87a34 <github.com/pingcap/tidb/pkg/server.(*clientConn).executePlanCacheStmt+0x00000000000001cc>  0x000000168509e3d8
0x00000003240d0788:  0x00000000065edd88  0x00000003e0f70d80
0x00000003240d0798:  0x00000000066020d8  0x00000003dcb4c000
0x00000003240d07a8:  0x0000000001baa410 <runtime.(*mspan).initHeapBits+0x0000000000000050>  0x0000000001baa410 <runtime.(*mspan).initHeapBits+0x0000000
000000050>
0x00000003240d07b8:  0x0000000001bbf98c <runtime.(*mheap).alloc+0x000000000000006c>  0x0000000005036a60
0x00000003240d07c8:  0x0000000000000000  0x0000000001baa410 <runtime.(*mspan).initHeapBits+0x0000000000000050>
0x00000003240d07d8:  0x0000000000000000  0x0000000001babde0 <runtime.(*mcentral).grow+0x0000000000000078>
0x00000003240d07e8:  0x00000003240d07f8  0x00000003dca35f80
0x00000003240d07f8:  0x0000000000000010  0x0000000000000010
fatal error: traceback stuck

goroutine 0 gp=0x3f48781880 m=11 mp=0x3f48796008 [idle]:
time.Now()
        /home/debian/go1.22.7/src/time/time.go:1132 +0x20 fp=0x3240d0298 sp=0x3240d0290 pc=0x1c365e0
time.Now()
        /home/debian/go1.22.7/src/time/time.go:1132 +0x20 fp=0x3240d02a0 sp=0x3240d0298 pc=0x1c365e0
github.com/pingcap/tidb/pkg/session.(*session).ExecuteStmt(0x9ae0fa0, {0x0?, 0x0?}, {0x4d88254, 0x4b1d96a08})
        /home/debian/rvspoc-S2427-tidb/pkg/session/session.go:2058 +0x448 fp=0x3240d0708 sp=0x3240d02a0 pc=0x4cbb330
github.com/pingcap/tidb/pkg/parser/ast.(*node).Text.func1()
        /home/debian/rvspoc-S2427-tidb/pkg/parser/ast/base.go:56 fp=0x3240d0708 sp=0x3240d0708 pc=0x1fd03a0
github.com/pingcap/tidb/pkg/session.(*session).ExecuteStmt.deferwrap1()
        /home/debian/rvspoc-S2427-tidb/pkg/session/session.go:2007 fp=0x3240d0708 sp=0x3240d0708 pc=0x4cbc040
runtime: traceback stuck. pc=0x4cbc040 sp=0x3240d0708
stack: frame={sp:0x3240d0708, fp:0x3240d0708} stack=[0x3f48798000,0x3f487a0000)

fatal error: traceback stuck
panic during panic

goroutine 0 gp=0x3f48781880 m=11 mp=0x3f48796008 [idle]:
time.Now()
        /home/debian/go1.22.7/src/time/time.go:1132 +0x20 fp=0x3240d0298 sp=0x3240d0290 pc=0x1c365e0
time.Now()
        /home/debian/go1.22.7/src/time/time.go:1132 +0x20 fp=0x3240d02a0 sp=0x3240d0298 pc=0x1c365e0
github.com/pingcap/tidb/pkg/session.(*session).ExecuteStmt(0x9ae0fa0, {0x0?, 0x0?}, {0x4d88254, 0x4b1d96a08})
        /home/debian/rvspoc-S2427-tidb/pkg/session/session.go:2058 +0x448 fp=0x3240d0708 sp=0x3240d02a0 pc=0x4cbb330
github.com/pingcap/tidb/pkg/parser/ast.(*node).Text.func1()
        /home/debian/rvspoc-S2427-tidb/pkg/parser/ast/base.go:56 fp=0x3240d0708 sp=0x3240d0708 pc=0x1fd03a0
github.com/pingcap/tidb/pkg/session.(*session).ExecuteStmt.deferwrap1()
        /home/debian/rvspoc-S2427-tidb/pkg/session/session.go:2007 fp=0x3240d0708 sp=0x3240d0708 pc=0x4cbc040
runtime: traceback stuck. pc=0x4cbc040 sp=0x3240d0708
stack: frame={sp:0x3240d0708, fp:0x3240d0708} stack=[0x3f48798000,0x3f487a0000)

fatal error: traceback stuck
stack trace unavailable
